test/unit/pcv/NamedStaticPCVDepositWrapper.test.ts

```diff
@@ -0,0 +1,302 @@
+  before(async () => {
+    const addresses = await getAddresses();
+
+    // add any addresses you want to impersonate here
+    const impersonatedAddresses = [addresses.governorAddress];
+
+    for (const address of impersonatedAddresses) {
+      impersonatedSigners[address] = await getImpersonatedSigner(address);
+    }
+  });
+
+  beforeEach(async function () {
+    ({ governorAddress } = await getAddresses());
+
+    newDepositName = 'Visor Finance Deposit';
+    balance = '2000';
+    feiBalance = '1000';
+    underlyingTokenAmount = '100000';
+    core = await getCore();
+    deposit = await (
+      await ethers.getContractFactory('NamedStaticPCVDepositWrapper')
+    ).deploy(core.address, [
+      {
+        depositName: newDepositName,
+        usdAmount: balance, /// USD equivalent in this deposit, not including FEI value
+        feiAmount: feiBalance, /// amount of FEI in this deposit
+        underlyingTokenAmount: underlyingTokenAmount, /// amount of underlying token in this deposit
+        underlyingToken: await core.fei()
+      }
+    ]);
+  });
+
+  describe('init', function () {
+    it('reported in USD', async function () {
+      expect(await deposit.balanceReportedIn()).to.be.equal('0x1111111111111111111111111111111111111111');
+    });
+
+    it('depositName', async function () {
+      const { depositName } = await deposit.pcvDeposits(0);
+      expect(depositName).to.be.equal(newDepositName);
+    });
+
+    it('usdAmount', async function () {
+      const { usdAmount } = await deposit.pcvDeposits(0);
+      expect(usdAmount).to.be.equal(balance);
+    });
+
+    it('underlyingTokenAmount', async function () {
+      const { underlyingTokenAmount } = await deposit.pcvDeposits(0);
+      expect(underlyingTokenAmount).to.be.equal(underlyingTokenAmount);
+    });
+
+    it('underlying', async function () {
+      const { underlyingToken } = await deposit.pcvDeposits(0);
+      expect(underlyingToken).to.be.equal(await core.fei());
+    });
+
+    it('feiAmount', async function () {
+      const { feiAmount } = await deposit.pcvDeposits(0);
+      expect(feiAmount).to.be.equal(feiBalance);
+    });
+
+    it('numDeposits', async function () {
+      expect(await deposit.numDeposits()).to.be.equal(1);
+    });
+
+    it('getAllUnderlying', async function () {
+      const allUnderlying = await deposit.getAllUnderlying();
+      expect(allUnderlying.length).to.be.equal(1);
+      expect(allUnderlying[0]).to.be.equal(await core.fei());
+    });
+
+    it('returns stored values', async function () {
+      expect(await deposit.balance()).to.be.equal(balance);
+      expect(await deposit.feiReportBalance()).to.be.equal(feiBalance);
+
+      const resistantBalances = await deposit.resistantBalanceAndFei();
+
+      expect(resistantBalances[0]).to.be.equal(balance);
+      expect(resistantBalances[1]).to.be.equal(feiBalance);
+    });
+  });
+
+  describe('addDeposit', function () {
+    it('add new deposit', async function () {
```

Comment:
"successfully adds a new deposit"

(again, entire test string should read like natural language)

Reply:
If you would like to make a change to our unit testing conventions, please suggest a change here and we can review as a team: https://docs.google.com/document/d/1iDjcVXTc8vM9TSLHLSQJU7ZB9MfhlCmRs7YF1Gnuc2Y/edit

Reply:
This isn't a change to our testing conventions; this is just good code style